src/components/profile/Profile.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Profile.scss';
+import MyListings from '../myListings/MyListings.js'
+import MyOffers from '../myOffers/MyOffers.js'
 
 const Profile = () => {
+
+  const [view, setView] = useState(true)
+
+  const handleClick = (e) => {
```

Comment:
I feel great about how dry and dynamic we made this!

Reply:
Yes, I agree. It is concise!